.storybook/config.js

```diff
@@ -1,17 +1,16 @@
 import React from 'react'
 import { configure, addDecorator } from '@storybook/react'
-import { Provider } from 'rebass'
-import { injectGlobal } from 'styled-components'
-import theme, { globalStyles } from '../src/layouts/rebass-theme'
-import backgroundColor from 'react-storybook-decorator-background'
-import 'normalize.css'
-import '../src/layouts/index.css'
+import { action } from '@storybook/addon-actions'
+import { ThemeProvider } from 'styled-components'
+// import { injectGlobal } from 'styled-components'
```

Comment:
@ric9176 can you please remove commented out code or add an explanation why we should keep it